src/lasdi/latent_space.py

```diff
@@ -1,175 +1,482 @@
+
+
+        -------------------------------------------------------------------------------------------
+        Arguments
+        -------------------------------------------------------------------------------------------
+
+        physics: A "Physics" object that holds the fom solution frames. We use this object to 
+        determine the shape of each fom solution frame. Recall that each Physics object has a 
+        corresponding PDE. We 
+
+        config: A dictionary representing the loaded .yml configuration file. We expect it to have 
+        the following keys/:
+            hidden_units: A list of integers specifying the dimension of the co-domain of each 
+            encoder layer except for the final one. Thus, if the k'th layer maps from 
+            \mathbb{R}^{n(k)} to \mathbb{R}^{n(k + 1)} and there are K layers (indexed 0, 1, ... , 
+            K - 1), then hidden_units should specify n(1), ... , n(K - 1). 
+
+            latent_dimension: The dimensionality of the Autoencoder's latent space. Equivalently, 
+            the dimensionality of the co-domain of the encoder (i.e., the dimensionality of the 
+            co-domain of the last layer of the encoder) and the domain of the decoder (i.e., the 
+            dimensionality of the domain of the first layer of the decoder).
+            
+
+        -------------------------------------------------------------------------------------------
+        Returns
+        -------------------------------------------------------------------------------------------
+
+        Nothing!
+        """
 
-    def __init__(self, physics, config):
         super(Autoencoder, self).__init__()
 
-        self.qgrid_size = physics.qgrid_size
-        self.space_dim = np.prod(self.qgrid_size)
-        hidden_units = config['hidden_units']
-        n_z = config['latent_dimension']
-        self.n_z = n_z
-
-        layer_sizes = [self.space_dim] + hidden_units + [n_z]
-        #grab relevant initialization values from config
-        act_type = config['activation'] if 'activation' in config else 'sigmoid'
-        threshold = config["threshold"] if "threshold" in config else 0.1
-        value = config["value"] if "value" in config else 0.0
-        num_heads = config['num_heads'] if 'num_heads' in config else 1
-
-        self.encoder = MultiLayerPerceptron(layer_sizes, act_type,
-                                            reshape_index=0, reshape_shape=self.qgrid_size,
-                                            threshold=threshold, value=value, num_heads=num_heads)
-        
-        self.decoder = MultiLayerPerceptron(layer_sizes[::-1], act_type,
-                                            reshape_index=-1, reshape_shape=self.qgrid_size,
-                                            threshold=threshold, value=value, num_heads=num_heads)
+        # A Physics object's qgrid_size is a list of integers specifying the shape of each frame of 
```

Comment:
Move these comments after 364 with `""" ... """` comment so that it can be parsed into API.

I personally think these belong to `Physics` class comments, but we can leave it here. It'd be great if we can refer readers to `Physics` class.